examples/hpc/hello-dla/Justfile

```diff
@@ -6,7 +6,13 @@ run-asic example *args:
         --config "target.riscv64imac-unknown-none-elf.runner = 'riscv32-unknown-elf-gdb -x connect-and-load.gdb'" \
         run -Fasic --example {{example}} {{args}}
 
+run-asic-64 example *args:
+    cargo \
+        --config "target.riscv64imac-unknown-none-elf.runner = 'riscv64-elf-gdb -x connect-and-load.gdb'" \
+        run -Fasic --example {{example}} {{args}}
+
```

Comment:
`run-asic-macos` might be the more appropriate name for this.

Reply:
Renamed just tasks using to brew versions of gdb as `*-*-macos`